apispec/api.spec.json

```diff
@@ -43,6 +43,36 @@
                     "services"
                 ],
                 "summary": "get a list of services a user is entitled to",
+                "parameters": [
+                    {
+                        "in": "query",
+                        "name": "include_bundles",
+                        "required": false,
+                        "description": "Filter response to only include listed bundles.",
+                        "schema":{
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                        },
+                        "explode": false,
+                        "style": "form"
+                    },
+                    {
+                        "in": "query",
+                        "name": "exclude_bundles",
+                        "required": false,
+                        "description": "Filter response to not include listed bundles.",
```

Comment:
Maybe we can add something like that `include_bundles` has priority when `exclude_bundles` is also present in request.

Reply:
That's a good point, will do!